cli/pcluster/config/mappings.py

```diff
@@ -524,8 +529,20 @@
                 "update_policy": UpdatePolicy.UNSUPPORTED
             }),
             ("auto_import_policy", {
+                "type": NoneCfnParam,
+                "NONE_value": "NONEDEFAULT",
                 "validators": [fsx_lustre_auto_import_validator],
-                "allowed_values": ALLOWED_VALUES["auto_import_policy"],
+                "allowed_values": ALLOWED_VALUES["fsx_auto_import_policy"],
+                "update_policy": UpdatePolicy.UNSUPPORTED
+            }),
+            ("storage_type", {
+                "allowed_values": ALLOWED_VALUES["fsx_storage_type"],
+                "update_policy": UpdatePolicy.UNSUPPORTED
+            }),
+            ("drive_cache_type", {
+                "type": NoneCfnParam,
+                "NONE_value": "NONEDEFAULT",
```

Comment:
let's rename it to `"cfn_default" :  "NONEDEFAULT",`

Reply:
We also need to fix all the parameter in the sections that have a "default" value but we want to convert to NONE when passed to CFN. 
In details the parameter on which we need to add `"cfn_default" : "NONE"` are: 
* EFS --> performance_mode, encrypted, throughput_mode
* RAID --> num_of_raid_volumes, volume_type, volume_size, volume_iops, encrypted
* DCV --> port, access_from